src/main/java/ch/jalu/configme/Comment.java

```diff
@@ -22,4 +22,12 @@
      */
```

Comment:
Update class Javadoc to mention that this annotation can be put on bean fields now